Grbl_Esp32/src/Settings.cpp

```diff
@@ -506,6 +514,7 @@ Error EnumSetting::setStringValue(char* s) {
             _storedValue = _currentValue;
         }
     }
+    check(NULL);
```

Comment:
The trello card is not really complete until the check(NULL) has been added to all the setStringValue methods and the various check* routines amended to check the value argument